### PR TITLE
New recipe: PhyML-v3.3.20210609

### PR DESCRIPTION
**Pull request by BenjaminDoran**

### CODE REVIEW

File: P/PhyML/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+# Make MPI version
+./configure ${extraflags} --enable-phyml-mpi --prefix=$prefix --build=${MACHTYPE} --host=${target}
+make clean & make -j${nproc}
+install -Dvm 755 ./src/phyml-mpi${exeext} ${bindir}/phymlMPI${exeext}
+"""
+
+platforms = supported_platforms()
+
+# similar error for below platforms
```
Comment (giordano) [inline P/PhyML/build_tarballs.jl:36]: https://github.com/JuliaPackaging/Yggdrasil/blob/9cee6a3185646db8d27fcc9fded39957e881bff1/F/FastME/build_tarballs.jl#L18-L22

File: P/PhyML/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    "aarch64-linux-musl", 
+    "armv6l-linux-musleabihf", 
+    "armv7l-linux-musleabihf",
+    "i686-linux-musl",
+    "aarch64-apple-darwin",
+    "x86_64-apple-darwin",
+    ]
+filter!(p->triplet(p) ∉ _rpl_malloc_error_triplets, platforms)
+
+# [16:08:21] /opt/i686-w64-mingw32/bin/../i686-w64-mingw32/sys-root/lib/../lib/libmingw32.a(lib32_libmingw32_a-crt0_c.o): In function `main':
```
Comment (giordano) [inline P/PhyML/build_tarballs.jl:61]: This isn't enough to tell what's happening, but I bet the build system is doing something dodgy.

File: P/PhyML/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+# [16:08:21] make[1]: Leaving directory '/workspace/srcdir/phyml'
+# [16:08:21] make[2]: *** [Makefile:1695: phyml.exe] Error 1
+# [16:08:21] make[1]: *** [Makefile:365: all-recursive] Error 1
+# [16:08:21] make: *** [Makefile:306: all] Error 2
+# [16:08:21]  ---> make -j${nproc}
+# [16:08:21]  ---> make -j${nproc}
+# [16:08:21] Previous command 1382 exited with 2
+filter!(p-> triplet(p) != "i686-w64-mingw32", platforms)
+
+# [15:56:47] i686-w64-mingw32-gcc  -std=c99 -O3 -fomit-frame-pointer -funroll-loops -Wall -Winline -finline    -o phyml.exe   -lm
```
Comment (giordano) [inline P/PhyML/build_tarballs.jl:74]: I have zero clue why the build system decided to use a non-existing compiler

File: P/PhyML/build_tarballs.jl [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    ExecutableProduct("phyml", :phyml),
+    ExecutableProduct("phymlMPI", :phymlMPI),
+]
+
+dependencies = [
+    Dependency(PackageSpec(name="MPICH_jll", uuid="7cb0a576-ebde-5e09-9194-50597f1243b4"); platforms=filter(!Sys.iswindows, platforms)),
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"); platforms=filter(!Sys.isbsd, platforms)),
+]
+
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies, julia_compat="1.6")
```
Comment (giordano) [inline P/PhyML/build_tarballs.jl:96]: Missing newline
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies, julia_compat="1.6")

```

File: P/PhyML/build_tarballs.jl (hunk 1 above)
Comment (BenjaminDoran) [reply P/PhyML/build_tarballs.jl:36]: I'll work on this and try out the changes you suggested. But my next commit will probably not be until tomorrow

**Pull request merged**